sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDBStateStoreProvider.scala

```diff
@@ -527,7 +546,7 @@ private[sql] class RocksDBStateStoreProvider
      * @param operationName - name of the store operation
      * @param colFamilyName - name of the column family
      */
-    private def verifyColFamilyCreationOrDeletion(
```

Comment:
intentional ?

Reply:
Yeah, so we could use it [here](https://github.com/apache/spark/pull/47524/files#diff-76a55d7a6f0b91a19366e01c919c615a4d13afc1c1c37b017afa9df1c1705218R72)